diff -Nur rxvt-unicode-9.21-orig/doc/Makefile.in rxvt-unicode-9.21/doc/Makefile.in
--- rxvt-unicode-9.21-orig/doc/Makefile.in	2014-12-31 14:09:25.000000000 +0000
+++ rxvt-unicode-9.21/doc/Makefile.in	2015-12-22 16:18:09.022931006 +0000
@@ -98,7 +98,7 @@
 	$(INSTALL_DATA) rxvtc.1.man $(DESTDIR)$(man1dir)/$(RXVTNAME)c.$(man1ext)
 	$(INSTALL_DATA) rxvtd.1.man $(DESTDIR)$(man1dir)/$(RXVTNAME)d.$(man1ext)
 	$(INSTALL_DATA) rxvt.7.man  $(DESTDIR)$(man7dir)/$(RXVTNAME).$(man7ext)
-	@TIC@ $(srcdir)/etc/rxvt-unicode.terminfo
+	@TIC@ -o$(DESTDIR)$(prefix)/share/terminfo $(srcdir)/etc/rxvt-unicode.terminfo
 
 distdepend: alldoc
 
